googlemock/test/gmock-matchers_test.cc

```diff
@@ -2778,7 +2778,7 @@ class FloatingPointTest : public testing::Test {
 
   // Pre-calculated numbers to be used by the tests.
 
-  const size_t max_ulps_;
+  const unsigned int max_ulps_;
```

Comment:
The latter. Leave it as a size_t and figure the warnings out as they occur. Why is there so much assignment of size_t to unsigned? Should those unsigned really be size_t?